TestMain.py

```diff
@@ -30,6 +43,9 @@ def test_get_repository_name(self):
         url = 'git@github.com:awslabs/aws-kms-compliance-checker.git'
         self.assertEqual(main.get_repository_name(url), 'aws-kms-compliance-checker')
 
+        url = 'https://github.com/tahiat/specimin.git'
```

Comment:
This should use the main Specimin repo (which is my fork: https://github.com/kelloggm/specimin).